src/controllers/trip.controller.js

```diff
@@ -116,6 +116,30 @@ class TripController {
     ResponseService.setSuccess(200, 'List of all available locations', availableLocations);
     return ResponseService.send(res);
   }
+
+  /**
+ *
+ *
+ * @static
+ * @param {req} req
+ * @param {res} res
+ * @returns {response} @memberof Trips
+ */
+  static async editOpenTripRequest(req, res) {
+    const data = req.userData, userId = data.id, { tripId } = req.params;
```

Comment:
I think you can rename this variable `data` to whatever data you need, for example, `userData`. you need to specify which data you need